packages/next/client/image.tsx

```diff
@@ -308,6 +309,10 @@ function imgixLoader({ root, src, width, quality }: LoaderProps): string {
   return `${root}${normalizeSrc(src)}${paramsString}`
 }
 
+function akamaiLoader({ root, src, width }: LoaderProps): string {
+  return `${root}${normalizeSrc(src)}${width ? '?imwidth=' + width : ''}`
```

Comment:
Does Akamai support an auto-format feature like [imgix](https://docs.imgix.com/apis/rendering/auto/auto#format) or [cloudinary](https://cloudinary.com/documentation/image_optimization#how_to_optimize_image_format)?

Reply:
Nope, as far as I can tell Akamai only supports width as a URL parameter (https://developer.akamai.com/api/web_performance/image_manager/v2.html). Quality is something the app developers will have to set using their own Akamai image policies.

I mentioned that in the description for this PR, but maybe I should add it as a comment in the code as well

Reply:
I see. I misunderstood "your side". Sounds like that means the "Akamai Dashboard" or something where its the same quality across all images?